packages/legend-application-studio/src/components/editor/editor-group/service-editor/testable/ServiceTestDataEditor.tsx

```diff
@@ -838,23 +868,23 @@ export const NewConnectionDataModal = observer(
                     options={dataElementOptions}
                     onChange={onDataElementChange}
                     value={selectedDataElement}
-                    darkMode={true}
+                    darkMode={
+                      !applicationStore.layoutService
+                        .TEMPORARY__isLightColorThemeEnabled
+                    }
                   />
                 </div>
               </div>
             )}
           </ModalBody>
           <ModalFooter>
-            <button
-              type="button" // prevent this toggler being activated on form submission
-              className="btn btn--dark"
+            <ModalFooterButton
               onClick={closeModal}
-            >
-              Cancel
-            </button>
-            <button className="btn btn--dark" disabled={isDisabled}>
-              Create
-            </button>
+              text="Cancel"
+              type="secondary"
+              preventFormSubmit={true} // prevent this toggler being activated on form submission
+            />
```

Comment:
This is the only change I haven't been able to test since I'm not familiar with how to get to the `ServiceTestDataEditor`. This component used to just use a `button` element with `type="button"` to prevent form submission; I added a `preventFormSubmit` property to the `ModalFooterButton` that, when true, sets the underlying button to also have `type="button"`, so it should work the same, but I think it would still be good to test that this component still operates as expected.